extra-optenvw64/mingw+w64/spec

```diff
@@ -1,4 +1,3 @@
-VER=5.0.3
-REL=2
+VER=8.0.0
 SRCTBL="https://sourceforge.net/projects/mingw-w64/files/mingw-w64/mingw-w64-release/mingw-w64-v$VER.tar.bz2"
-CHKSUM="sha256::2a601db99ef579b9be69c775218ad956a24a09d7dabc9ff6c5bd60da9ccc9cb4"
```

Comment:
Please use the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/).